semantic-conventions/src/opentelemetry/semconv/model/semantic_attribute.py

```diff
@@ -192,7 +193,7 @@ def parse(
                 )
                 msg = f"Semantic convention stability set to deprecated but attribute '{attr_id}' is {stability}"
                 raise ValidationError.from_yaml_pos(position, msg)
-            stability = stability or semconv_stability or StabilityLevel.STABLE
+            stability = stability or semconv_stability or StabilityLevel.EXPERIMENTAL
```

Comment:
This seems like a good change, but also important enough to include in the PR title